packages/@react-spectrum/s2/stories/Tabs.stories.tsx

```diff
@@ -99,6 +99,10 @@ export const Icons = (args: any) => (
   </div>
 );
 
+export const Icons = {
+  render: (args) => <IconsRender {...args} />
```

Comment:
can we set iconOnly to true for this story? little confusing that the story is called icons but the tabs first render with icons + text

Reply:
Don't we want a story with icons and text, and we set the `iconOnly` prop to remove the text?

Reply:
yeah i suppose that's true. i guess we can leave it as it is then